earthpy/tests/test_plot.py

```diff
@@ -88,3 +87,29 @@ def test_single_band_2dims():
     assert arr.ndim == 2
     assert len(fig.axes[0].get_images()) == 1
 
+""" Colorbar Tests """
+
+
+im_arr2 = np.random.randint(10, size=(5, 5))
+fig, ax = plt.subplots(figsize=(5, 5))
+im = ax.imshow(im_arr2,
+               cmap='RdYlGn')
+
+def test_colorbar_height():
```

Comment:
This test looks like it's written for a user-facing function with the try/except logic, but an `assert` statement would be more typical for a pytest test, e.g., 

```python
def test_colorbar_height():
    """Test that the colorbar ax height matches the image axis height"""
    fig, ax = plt.subplots(figsize=(5, 5))
    im = ax.imshow(np.random.randint(10, size=(5, 5)))
    cb = es.colorbar(im)
    assert cb.ax.get_position().height == im.axes.get_position().height
```

Reply:
i have been writing too many front end functions apparently. i have no idea what i was thinking. fixed.